lib-multisrc/madtheme/src/eu/kanade/tachiyomi/multisrc/madtheme/MadTheme.kt

```diff
@@ -177,30 +174,45 @@ abstract class MadTheme(
     }
 
     override fun chapterListParse(response: Response): List<SChapter> {
-        if (response.code in 200..299) {
+        if (response.request.url.fragment == "idFound") {
```

Comment:
you return this `idFound` in `chapterListRequest` everytime, so when would the later code be called/necessary?